web/src/message_store.ts

```diff
@@ -19,7 +19,7 @@ export type DisplayRecipientUser = {
     email: string;
     full_name: string;
     id: number;
-    is_mirror_dummy: boolean;
+    is_mirror_dummy?: boolean;
```

Comment:
It doesn't seem like this is used at all in the frontend, so maybe we want to remove it completely here?

Reply:
Yeah, looks like we can avoid sending it to the client to reduce response data. @timabbott FYI

Reply:
@evykassirer I think you should start a czo discussion to fast track that.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/remove.20is_mirror_dummy

Reply:
It seems like just leaving it there is fine then, so I'll do that.